apps/web/src/hooks/infinity/useInfinityCakeAPR.ts

```diff
@@ -153,7 +152,7 @@ export const useInfinityBinPositionCakeAPR = ({
       ? Object.values(rewardsPerEpoch).reduce((acc, r) => acc.plus(r), BIG_ZERO)
       : undefined
 
-    const rewardForPositionPerYear = positionRewardPerEpoch?.gt(0)
+    const rewardForPositionPerYear = positionRewardPerEpoch
```

Comment:
positionRewardPerEpochs comes 0 when non active items filtered, so it should be filtered based on undefined